poutyne/framework/callbacks/logger.py

```diff
@@ -1,17 +1,18 @@
-import os
 import csv
+import os
+from typing import Dict, TextIO, BinaryIO, Union
```

Comment:
I don't think the file descriptors can `TextIO` in our case since we're saving binary.

Reply:
Was not sure if where also using TextIO.

Reply:
will remove